tidb-cloud/tidb-cloud-billing.md

```diff
@@ -112,13 +112,24 @@ To view the billing details, perform the following steps:
 2. Click **Billing**.
 3. Click **Bills**. The billing details page is displayed.
 
-## Trial points
+## Credits
 
-During the [PoC](/tidb-cloud/tidb-cloud-poc.md) period, you can use trial points to pay the TiDB cluster fees. One point is equivalent to one U.S. dollar. All your TiDB clusters will be automatically terminated when the trial points are used up.
+TiDB Cloud offers a certain number of credits for Proof of Concept (PoC) users. One credit is equivalent to one U.S. dollar. You can use credits to pay TiDB cluster fees before the credits become expired.
+
+Different types of credits can be used to pay different fees.
```

Comment:
Currently, only the "General" type is supported.

Reply:
@yuezhanglang PTAL

Reply:
I have deleted this sentence.